### PR TITLE
implement in server and front end

### PR DESCRIPTION
**Pull request by kendall-butt**

### CODE REVIEW

File: client/Components/ProductInfo.jsx [hunk 1]
```diff
@@ -8,20 +8,31 @@ export default class ProductInfo extends React.Component {
...
 
   componentDidMount() {
-    this.setState({
-      productId: [window.location.href.split('/')[4]],
-    }, () => {
-      fetch(`http://localhost:3002/products/${this.state.productId}/description-and-standards`).then((res) => res.json()).then((data) => this.setState({
-        productInfo: data,
-      })).catch((err) => console.log(err));
-    });
+    const param = [window.location.href.split('/')[4]];
```
Comment (Tina-Luk) [inline client/Components/ProductInfo.jsx:18]: Is search term and product Id always the same value?

Comment (Tina-Luk) [inline client/Components/ProductInfo.jsx:18]: Is search term and product Id always the same value?

File: index.js [hunk 2]
```diff
@@ -61,6 +61,43 @@ app.get('/products/:Id/description-and-standards', (req, res) => {
   });
 });
 
+app.get('/products/:searchTerm/description', (req, res) => {
```
Comment (Tina-Luk) [inline index.js:64]: something to consider to modularize code. All the code is inside index.js - maybe add a controller and/or model file to modularize?

**Review by Tina-Luk — APPROVED** [review]
Looks good, just a few suggestions!

File: client/Components/ProductInfo.jsx (hunk 1 above)
Comment (kendall-butt) [reply client/Components/ProductInfo.jsx:18]: No, search term would be a string and product id would be the unique identifier of a product. This is set up to allow for both, but the front end doesn't accommodate both (yet)

**Pull request merged**